be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -178,6 +178,10 @@ bool ScanOperator::has_output() const {
         }
     }
 
+    if (_morsel_queue->has_more()) {
```

Comment:
Is there a problem with returning false directly without check has_more?

Reply:
you are correct.  it's a bug